src/main/java/site/coach_coach/coach_coach_server/routine/services/RoutineService.java

```diff
@@ -39,6 +44,10 @@ public RoutineListCoachInfoDto getRoutineListCoachInfo(RoutineListRequest routin
 		return routineListCoachInfoDto;
 	}
 
+	public Long getCoachId(Long userIdByJwt) {
+		return userRepository.findById(userIdByJwt).get().getCoach().getCoachId();
```

Comment:
여기서 에러 처리 해주는 게 좋을 것 같아요

```suggestion
		return userRepository.findById(userIdByJwt)
            .orElseThrow(() -> new UserNotFoundException(ErrorMessage.USER_NOT_FOUND))
            .getCoach()
            .getCoachId();
```

음 그리고 findById라는 함수가 있나요????? findByUserId아니었나요? id에 대한 명시가 있으면 더 좋을 것 같습니다!!

Reply:
findById는 해당 Repository의 Entity에서 @Id를 가진 값으로 찾을 수 있는 JPA 기본 제공함수입니다!

Reply:
그리고 메서드 명도 `getCoachIdByUserId`로 바꾸는 건 어떨까용??? 사용자의 코치 아이디를 반환하는 거니까요!

Reply:
근데 findById로만 하면 어떤 아이디로 가져오는 지 조금.. 헷갈릴 것 같긴합니당 userId로 가져오는 거면 findByUserId로 쓰는 것이 좋지 않을까요?

Reply:
findById를 제공하는지는 몰랐네용... 하나 배워갑니다